src/signals/incident-management/components/FilterForm/__tests__/FilterForm.test.js

```diff
@@ -258,7 +258,10 @@ describe('signals/incident-management/components/FilterForm', () => {
 
     expect(submitButton.textContent).toEqual(DEFAULT_SUBMIT_BUTTON_LABEL);
 
-    act(() => { fireEvent.change(nameField, { target: { value: 'My filter' } }); });
+    act(() => {
+      fireEvent.change(nameField, { target: { value: 'My filter' } });
+      fireEvent.blur(nameField);
+    });
```

Comment:
If both `onChange` and `onBlur` events have to stay in place, then please separate their behaviour into different test cases instead of combining them into the same `act` call. It's impossible now to determine which handler updated the state.
That also goes for tests below.

Reply:
Thanks, we indeed don't need the `onchange fireEvents` anymore :)